mypy/typeops.py

```diff
@@ -8,11 +8,12 @@
 # TODO: Move more type operations here
 
 from mypy.types import TupleType, Instance
-from mypy.join import join_type_list
 
 
 def tuple_fallback(typ: TupleType) -> Instance:
     """Return fallback type for a tuple."""
+    from mypy.join import join_type_list
```

Comment:
What's this change?

Reply:
There is some fragility in mypy about import cycles, where certain modules need to be imported before others for things to work. Removing references to the old analyzer caused one of these places to surface when trying to run tests, so I fixed it. There are a couple more that I will try to fix in a separate PR.

Reply:
One of these days I'm going to try to clean up the import cycles (though it might not help with this in particular).